src/DurableTask.SqlServer.AzureFunctions/SqlMetricsProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.SqlServer.AzureFunctions
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class SqlMetricsProvider
+    {
+        readonly SqlOrchestrationService service;
+
+        public SqlMetricsProvider(SqlOrchestrationService service, int? previousWorkerCount = null)
```

Comment:
is `previousWorkerCount` unused? Do we need it in the constructor at all?

Reply:
Removed.